Header header
int16 reward_tom
int16 reward_jerry
int16 iteration_num